x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph_investigation/graph_investigation.stories.test.tsx

```diff
@@ -344,39 +345,64 @@ describe.skip('GraphInvestigation Component', () => {
         query: '(host1) OR event.id: "1" OR event.id: "2"',
         language: 'kuery',
       });
-      expect(onInvestigateInTimeline.mock.calls[0][FILTERS_PARAM_IDX]).toEqual([
-        {
-          $state: {
-            store: 'appState',
+      expect(onInvestigateInTimeline.mock.calls[0][FILTERS_PARAM_IDX]).toEqual([]);
+    });
+
+    it('empty query and empty originEventIds - calls onInvestigateInTimeline with empty filters', () => {
```

Comment:
fyi, `empty originEventIds` is not a scenario we have today so it is less relevant right now
but good to have it :)